### DISCUSSION TITLE
Absolute Paths for Templates?

### DISCUSSION THREAD

**Discussion by DevOpsBoondoggles** [PowerShellOrg/Plaster#399] [Ideas]
Not sure if I'm being stupid here but can I not use an absolute path for a template ?
e.g. 

`<file destination='src/${PLASTER_PARAM_ModuleName}.psm1' source='${PLASTER_PARAM_TemplatesFolderPath}\template.psm1' />`

where ${PLASTER_PARAM_TemplatesFolderPath} goes to \\Server\fileshare\ 

The error: 
"template.psm1' specified in the file directive in the template manifest cannot be an absolute path.  
Change the path to a relative path."  

Which seems really strange. My use case is that I am sharing a module that creates other modules for a company, and I want to reference a shared template in the area.

I'm going to work around by copying it local but wasn't sure if it was just a bad error message and I'm missing something obvious.    I did look around but I didn't see anything related, though I could just be being dumb.

**Comment by rkeithhill** [comment]
This is limitation that we might be able to lift. The intent is to prevent a Plaster template from **modifying** a directory other than the dir that you're applying the template too.  However, copy file from another location into the template source dir seems like that might be a safe thing to do.

**Comment by DevOpsBoondoggles** [comment]
I am probably too rubbish at Powershell to assist with this but happy to try if someone could point me where to look
